pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -117,125 +119,134 @@ type Cluster struct {
+	// `Events` based on several configured buckets.
 	EventRateLimitConfig *EventRateLimitConfig `json:"eventRateLimitConfig,omitempty"`
 
-	// AdmissionPlugins provides the ability to pass arbitrary names of admission plugins to kube-apiserver
-	AdmissionPlugins []string `json:"admissionPlugins,omitempty"`
+	// Optional: Deploys the UserSSHKeyAgent to the user cluster.
+	// If enabled, the agent will be deployed and used to sync user ssh keys attached by users to the cluster.
+	// No SSH keys will be synced after node creation if this is disabled. Once the agent is enabled/disabled
+	// it cannot be changed after cluster creation.
+	EnableUserSSHKeyAgent *bool `json:"enableUserSSHKeyAgent,omitempty"`
+
+	// Optional: Enables operating-system-manager (OSM), which is responsible for creating and managing worker node configuration.
+	// This is an experimental feature.
+	EnableOperatingSystemManager bool `json:"enableOperatingSystemManager,omitempty"`
+
+	// KubernetesDashboard holds the configuration for the kubernetes-dashboard component.
+	KubernetesDashboard KubernetesDashboard `json:"kubernetesDashboard,omitempty"`
 
+	// Optional: AuditLogging configures Kubernetes API audit logging (https://kubernetes.io/docs/tasks/debug-application-cluster/audit/)
+	// for the user cluster.
 	AuditLogging *AuditLoggingSettings `json:"auditLogging,omitempty"`
 
-	// OPAIntegration is a preview feature that enables OPA integration with Kubermatic for the cluster.
-	// Enabling it causes gatekeeper and its resources to be deployed on the user cluster.
+	// Optional: OPAIntegration is a preview feature that enables OPA integration for the cluster.
+	// Enabling it causes OPA Gatekeeper and its resources to be deployed on the user cluster.
 	// By default it is disabled.
 	OPAIntegration *OPAIntegrationSettings `json:"opaIntegration,omitempty"`
 
-	// ServiceAccount contains service account related settings for the kube-apiserver of user cluster.
+	// Optional: ServiceAccount contains service account related settings for the user cluster's kube-apiserver.
 	ServiceAccount *ServiceAccountSettings `json:"serviceAccount,omitempty"`
 
-	// MLA contains monitoring, logging and alerting related settings for the user cluster.
+	// Optional: MLA contains monitoring, logging and alerting related settings for the user cluster.
 	MLA *MLASettings `json:"mla,omitempty"`
 
-	// +kubebuilder:validation:Enum=docker;containerd
-	// +kubebuilder:default=containerd
-
-	// ContainerRuntime to use, i.e. Docker or containerd. By default containerd will be used.
-	ContainerRuntime string `json:"containerRuntime,omitempty"`
+	// Pause tells that this cluster is currently not managed by the controller.
+	// It indicates that the user needs to do some action to resolve the pause.
+	// +kubebuilder:default=false
+	Pause bool `json:"pause"`
+	// PauseReason is the reason why the cluster is no being managed.
```

Comment:
_not_ being managed.

Reply:
Also please add "This is purely informational and to be set by the user who pauses the cluster" or something along those lines.